lms/templates/ccx/enrollment.html

```diff
@@ -3,17 +3,17 @@
 <div class="batch-enrollment" style="float:left;width:50%">
   <form method="POST" action="ccx_invite">
   <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }">
-  <h2> ${_("Batch Enrollment")} </h2>
+  <h2 role="heading" tabindex="0"> ${_("Batch Enrollment")} </h2>
```

Comment:
According to Mark's comment on https://github.com/edx/edx-platform/pull/7895#discussion-diff-30075219 you should remove this role attribute.